src/main/java/com/zaxxer/nuprocess/linux/ProcessEpoll.java

```diff
@@ -178,9 +184,9 @@ public void closeStdin(LinuxProcess process)
    @Override
    public boolean process()
    {
-      Integer stdinFd = null;
-      Integer stdoutFd = null;
-      Integer stderrFd = null;
```

Comment:
Using `Integer` and doing a `null` check allocates 3 `Integer`s per (very tightly looped) `process` call. Individually, they don't matter much. But when pumping a large amount of data across a large number of processes, they contribute noticeably to GC churn. Using an `int` with a marker value helps avoid those allocations _on many `process` calls_ but not on all of them.

* `ident` is still boxed to call `fildesToProcessMap.get`
* All three descriptors are boxed if the process becomes eligible for soft-exit cleanup